test/examples/full-form.json

```diff
@@ -126,16 +133,48 @@
                   "default": "preferred_username",
                   "jsonPath": "$.spec.auth.sso.principalAttribute",
                   "description": "RH-SSO Principal Attribute to use as username"
+                },
+                {
+                  "label": "AMQ password",
+                  "type": "password",
+                  "jsonPath": "$.spec.commonConfig.amqPassword",
+                  "description": "The password to use for AMQ user. Generated if empty"
+                },
+                {
+                  "label": "AMQ cluster password",
+                  "type": "password",
+                  "jsonPath": "$.spec.commonConfig.amqClusterPassword",
+                  "description": "Password for AMQ Cluster. Generated if empty"
+                },
+                {
+                  "label": "Controller password",
+                  "type": "password",
+                  "jsonPath": "$.spec.commonConfig.controllerPassword",
+                  "description": "The password to use for 'controllerUser'. Generated if empty"
+                },
+                {
+                  "label": "Server password",
+                  "type": "password",
+                  "required": true,
```

Comment:
Just confirming that this is in fact correct, as the server password needs to be known by business central in order to access the kie server.